Kickstarter-iOS/Views/SettingsNotificationPickerCell.swift

```diff
@@ -0,0 +1,57 @@
+import Foundation
+import KsApi
+import Library
+import Prelude
+
+protocol SettingsNotificationPickerCellDelegate: class {
+  func didFailToSaveChange(errorMessage: String)
```

Comment:
I made a few comments in #403 too on the delegate method naming. It's up to you guys really but I tend to think the delegate method should include the object that triggered it. I know it can get a little verbose sometimes but it helps to stick to a convention. The [Google Swift style guide](https://google.github.io/swift/#delegate-methods) also explains it well.

So for this for example: 
```swift
func settingsNotificationPickerCell(_ cell: SettingsNotificationPickerCell, didFailToSaveChange errorMessage: String)
```

Reply:
Oh actually this delegate function isn't even needed. Will remove, but I hear you about delegate function naming.

Reply:
I'll rename the others